src/components/RepositoryContributionsCard.tsx

```diff
@@ -13,58 +13,86 @@ export const RepositoryContributionsCard = ({
-          {nodes?.map(({ pullRequest: { state, title, id, url } }: any) => (
-            <div key={id} className="flex items-center justify-between gap-2">
-              <a href={url} target="_blank" className="truncate" title={title}>
-                {title}
-              </a>
-              <span
-                className={`h-fit rounded p-1 ${
-                  state === "MERGED"
-                    ? "bg-purple-500"
-                    : state === "CLOSED"
-                    ? "bg-red-500"
-                    : "bg-green-500"
-                }`}
+    <div className="card bg-base-100 overflow-visible">
+      <div className="sm:w-auto card bg-base-100 repository-card">
+        <div className="card-body">
+          <div className="card-title flex items-center justify-between">
+            <div className="flex items-center space-x-2 flex-grow">
+              <Image
+                src={repository.owner.avatarUrl}
+                alt={repository.owner.login}
+                width={40}
+                height={40}
+                className="rounded-full"
+              />
+              <div
+                className="grid grid-flow-col gap-2 flex-grow tooltip text-left"
+                data-tip={`${repository.owner.login}/${repository.name}`}
               >
-                {state === "MERGED" ? (
-                  <FaCodeMerge size={18} />
-                ) : state === "CLOSED" ? (
-                  <IoIosCloseCircleOutline size={18} />
-                ) : (
-                  <GoIssueOpened size={18} />
-                )}
-              </span>
+                <Link
+                  href={`https://github.com/${repository.owner.login}/${repository.name}`}
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  className="hover:underline truncate flex-grow"
+                  aria-label={`${repository.name}`}
+                >
+                  <h3 className="logged-user truncate">
+                    {repository.owner.login}/{repository.name}
+                  </h3>
+                </Link>
+              </div>
+              <div
+                className="tooltip tooltip-left"
+                data-tip="Total contributions"
+              >
+                <div className="rounded outline outline-1 cursor-default px-2">
+                  {totalCount}
+                </div>
+              </div>
             </div>
-          ))}
+          </div>
+          <div className="max-h-[22rem] hide-scrollbar overflow-auto flex flex-col gap-1">
+            {nodes?.map(
+              (
+                { pullRequest: { state, title, id, url } }: any,
+                index: number
+              ) => (
+                <div
+                  key={id}
+                  className={`flex items-center justify-between gap-2 tooltip text-left ${
+                    index === 0 ? "tooltip-bottom" : "tooltip-top"
+                  }`}
+                  data-tip={title}
```

Comment:
This is a workaround for correctly displaying the tooltips.
If it is set to` tooltip-top` the first item's tooltip doesn't show, as well as if it is set to  `tooltip-bottom` the last one doesn't appear either. Feel free to discard these lines if you know a cleaner way to achieve the correct displaying of the tooltips.

Reply:
Ouch, I didn't notice it. And it makes sense it doesn't work. We need to find another way :thinking:
@CBID2 you can take over this if you like